.ci_support/linux_64_c_compiler_version10cdt_namecos6cuda_compiler_versionNonecxx_compiler_version10.yaml

```diff
@@ -20,6 +20,8 @@ docker_image:
 - quay.io/condaforge/linux-anvil-cos7-x86_64
 target_platform:
 - linux-64
+ucx:
+- 1.12.1
```

Comment:
Should this and others below be `1.13.0`?

Reply:
No these are unused. Downstream feedstocks would use this for determining what `ucx` version to use in `requirements/host` if `ucx` is listed there. As that doesn't occur here, it is unused.

This value is coming from [`conda-forge-pinning` as `ucx` is pinned to this version currently]( https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/865eeb7448f6fc35ebea69fbe602d2cea8342d52/recipe/conda_build_config.yaml#L784-L785 ). Once 1.13.0 is released, a migrator PR will be issued against `conda-forge-pinning` to start updating the version ( here's an example using `nettle`, https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/2955 ), then all feedstocks using `ucx` will be migrated in topological order ( here's an example of a migrator PR with `nettle`, https://github.com/conda-forge/gnutls-feedstock/pull/32 ), and after that the version would be updated in `conda-forge-pinning` ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/3058 ).

Reply:
Got it, thanks for the detailed explanation!